sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/TimeSeriesInsightsLifecycleSamples.cs

```diff
@@ -42,11 +42,8 @@ public async Task RunSamplesAsync()
 
                 #region Snippet:TimeSeriesInsightsUpdateModelSettings
```

Comment:
You might want to update the snippet to by more explicit that it's updating model settings by name.